common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -113,9 +112,6 @@ public final class PlatformDependent {
     private static final boolean IS_IVKVM_DOT_NET = isIkvmDotNet0();
 
     private static final int ADDRESS_SIZE = addressSize0();
-    private static final boolean USE_DIRECT_BUFFER_NO_CLEANER;
-    private static final AtomicLong DIRECT_MEMORY_COUNTER;
-    private static final long DIRECT_MEMORY_LIMIT;
```

Comment:
why is this removed ?

Reply:
These were refactored in the pr https://github.com/sirensolutions/netty/pull/26 to lazy load direct memory limit to ensure that the static fields will not be initialized before the creation of the siren plugin.

Reply:
thanks